### PR TITLE
Recursively get all dropdown items to support use of wrapper components

### PR DESCRIPTION
**Pull request by HeleneGrini**

### CODE REVIEW

File: component-lib/src/molecules/Dropdown/Dropdown.stories.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ const countries = [
   { name: 'Zimbabwe' },
 ];
 
-storiesOf('Business/Dropdown', module)
+storiesOf('Component library|Molecules/Dropdown', module)
```
Comment (nmenglund) [inline component-lib/src/molecules/Dropdown/Dropdown.stories.tsx:68]: Always use `/` as separator in Storybook 6.0 -- i.e. `Component library/Molecules/Dropdown`

File: component-lib/src/molecules/Dropdown/DropdownItem.tsx [hunk 2]
```diff
@@ -55,6 +53,12 @@ export const DropdownItem: React.FC<DropdownItemProps> = props => {
   const { open, toggle, highlightIndex } = useDropdownContext();
   const itemRef = useRef<HTMLButtonElement>(null);
 
+  useEffect(() => {
+    if (props.index === highlightIndex && itemRef.current) {
+      itemRef.current.scrollIntoView({ block: 'nearest', inline: 'nearest' });
```
Comment (nmenglund) [inline component-lib/src/molecules/Dropdown/DropdownItem.tsx:58]: Tested this in IE11? It seems IE11 only supports the boolean parameter type: https://caniuse.com/#search=scrollIntoView

Comment (nmenglund) [reply component-lib/src/molecules/Dropdown/DropdownItem.tsx:58]: (do we still need to support IE11? maybe put all IE11 polyfills a bucket script somewhere?)

**Review by nmenglund — CHANGES_REQUESTED** [review]

File: component-lib/src/molecules/Dropdown/useAccessibleDropdown.ts [hunk 3]
```diff
@@ -42,15 +42,17 @@ export const useAccessibleDropdown = () => {
       }
     };
     const navigate = (e: KeyboardEvent) => {
+      const arrowDown = 40;
+      const arrowUp = 38;
       if (ctx.open && e.target && ctx.menuRef.current) {
-        if (e.key === 'ArrowDown') {
+        if (e.keyCode === arrowDown) {
```
Comment (nmenglund) [inline component-lib/src/molecules/Dropdown/useAccessibleDropdown.ts:48]: I think the `const` definition is unnecessary; a `// arrow down` comment would suffice

File: component-lib/src/molecules/Dropdown/DropdownItem.tsx [hunk 4]
```diff
@@ -53,6 +53,13 @@ export const DropdownItem: React.FC<DropdownItemProps> = props => {
   const { open, toggle, highlightIndex } = useDropdownContext();
   const itemRef = useRef<HTMLButtonElement>(null);
 
+  /**
+   * Work in progress
+   *
+   * Currently works for dropdown menus without search item(s).
+   * NB: Not supported properly in ie11 from the styleguiden. This currently
+   * needs to be handled by the user
```
Comment (nmenglund) [inline component-lib/src/molecules/Dropdown/DropdownItem.tsx:61]: I think you should elaborate a little bit in the comment; _what_ isn't supported? _How_ do users need to handle it?

("NB: `scrollIntoView` is not supported properly in IE11`", "needs to be handled by the user with a polyfill")

**Review by nmenglund — CHANGES_REQUESTED** [review]

**Review by nmenglund — APPROVED** [review]

**Review by antons1 — APPROVED** [review]

**Pull request merged**